### ISSUE TITLE
Підвищити безпеку інсталяції шляхом перенесення публічних елементіів в підкаталог

### ISSUE THREAD

**Issue by yukoff** [hurtom/toloka#88]
В ідеалі все, що публічно доступно кладемо в підкаталог `web`, тоді `.htaccess` не потрібен, і має працювати однаково для apache та nginx

**Comment by yukoff** [comment]
Наприклад, в існуючій інсталяції можливо зробити таке `https://toloka.local/vendor/autoload.php` - воно нічого не верне, але зміст я думаю зрозумілий

**Comment by AucT** [comment]
або назвати public
кому як подобається.

**Comment by yukoff** [comment]
@AucT Точно, оскільки upstream рухаються у напрямку laravel, то `public` там типово. `web` то я за звичкою з symfony.

**Comment by konfuciusu** [comment]
Що стосується `vendor`, може є можливість налаштувати Composer так, щоб використовувалася папка на рівень вище?

Якщо говорити не тільки про `vendor`, було би добре скоординуватися із розробниками torrentpier.

**Comment by yukoff** [comment]
@konfuciusu Та тут мова не про `vendor/`, а про все, що в корені лежить (`*.php`, `admin/`, `bt/` тощо) - вони виходить як раз будуть читати bootstrap/autoload/etc з каталогу на рівень вище, але сам веб-сервер туди не дістанеться (я маю на увазі через GET/POST/... - php свої include-и прочитає без проблем). Такий собі ізольований webroot. А щодо upstream-у - тут я повністю згоден, наче з язика )
